src/lang/en.json

```diff
@@ -29,6 +29,7 @@
     "common.copy": "Copy",
     "common.createAccount": "Create Account",
     "common.dismiss": "Dismiss",
+    "common.error.clientConstruction": "Could not construct client to login",
```

Comment:
That's not a message I would want to surface to someone. How could this happen?

Reply:
I could just make it not do this instead? 

this happens when they try to login, but their wallet or client is null, where we just attempted to construct the client.